PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -559,26 +559,25 @@ protected function wait_channel($channel_id, $timeout = 0)
                 }
                 continue;
 
-            } else {
```

Comment:
Hui it's this part: https://github.com/videlalvaro/php-amqplib/blob/master/PhpAmqpLib/Connection/AbstractConnection.php#L560-L562

In the above `if` you use `continue` so the following `else is not needed